docs/time/index.rst

```diff
@@ -839,18 +915,19 @@ Examples::
 In this process the ``format`` and other object attributes like ``lon``,
 ``lat``, and ``precision`` are also propagated to the new object.
 
+.. EXAMPLE END
 
 As noted in the ``Time object basics`` section, a |Time| object is immutable and
```

Comment:
This is also no longer true. Maybe, "a |Time| object can only be changed by explicitly setting some of its elements. The process..."